docs.html

```diff
@@ -0,0 +1,552 @@
+            </p></p></code></pre>
+        </div>
+
+        <!--Avatar-->
+
+        <h2 id="avatar">Avatar</h2>
+        <p class="paragraph-text">Documentation and Examples on BuilderUI Alerts</p>
+        <h4>Examples</h4>
+        <div class="div-example">
+            <img class="avatar" src="https://via.placeholder.com/50">
+            <img class="avatar-small" src="https://via.placeholder.com/50">
+        </div>
+        <h4>Code Snippet</h4>
+        <div class="code-snippet">
+            <pre>
+                <code>
+        &ltimg class="avatar" src="https://via.placeholder.com/50&gt
+        &ltimg class="avatar-small" src="https://via.placeholder.com/50"&gt
+                </code>
+            </pre>
+        </div>
+
+        <!--Badge-->
+        <h2 id="badge">Badge</h2>
+        <h4>Examples</h4>
+        <div class="div-example">
+            <span class="badge badge-default">default</span>
+            <span class="badge badge-success">success</span>
+            <span class="badge badge-error">error</span>
+            <span class="badge badge-new">default</span>
+            <span class="badge badge-notification">5</span>
+
+            <h4>
+                Badge on Icons/Avatars
+            </h4>
+            <button class="icon-button">
+                <span class="material-icons">
+                    notifications
+                </span>
+                <span class="badge-notification2">25</span>
+            </button>
+
+        </div>
+        <h4>Code Snippet</h4>
+        <div class="code-snippet">
+            <pre>
+                <code>
+            &ltspan class="badge badge-default"&gtdefault&lt/span&gt
+            &ltspan class="badge badge-success"&gtsuccess&lt/span&gt
+            &ltspan class="badge badge-error"&gterror&lt/span&gt
+            &ltspan class="badge badge-new"&gtdefault&lt/span&gt
+            &ltspan class="badge badge-notification"&gt5&lt/span&gt
+            &ltbutton class="icon-button"&gt
+            &ltspan class="material-icons"&gt
+                notifications
+            &lt/span&gt
+            &ltspan class="badge-notification2"&gt25&lt/span&gt
+            &lt/button&gt
+                </code>
+            </pre>
+        </div>
+
+        <!--Button-->
+        <h2 id="button">Button</h2>
+        <h4>Examples</h4>
+        <div class="div-example">
+            <button class="btn btn-primary">Button</button>
+            <button class="btn btn-secondary">Button</button>
+            <button class="btn btn-ghost">Button</button>
+            <button class="btn btn-primary btn-icon">
+                <span class="material-icons">
+                    mail
+                </span>
+                <span>Email</span>
+            </button>
+            <button class="btn btn-secondary btn-icon">
+                <span class="material-icons">
+                    call
+                </span>
+                <span>Call Us</span>
+            </button>
```

Comment:
Use `inline-flex` instead of `flex` so that the buttons don't span the entire width of the container.